packages/web-components/@sfx/autocomplete/README.md

```diff
@@ -4,16 +4,42 @@
 
 The Autocomplete component displays search terms related to a query term. It also dispatches events whenever these search terms are interacted with.
 
-This component listens for and dispatches a number of events. These events are defined in the [`@sfx/events`][sfx-events] package.
+### Term Selection
+
+At most one term is selected at any given time.
+When a term is selected, the `aria-selected` property for the item is set to `true`.
+
+The `selectedIndex` property determines which item is selected;
+this value is the 0-based index of the selected item relative to the first item of the entire results.
+This property can be set to change the selection.
+If the value is outside the range of the results, no item will be selected.
```

Comment:
sorry did we want to mention that 'outside the results' includes a negative value? @jmbtutor

Reply:
I can add that in.